packages/web-app-files/src/components/SideBar/Shares/Links/DetailsAndEdit.vue

```diff
@@ -67,6 +67,7 @@
       />
     </p>
     <div :class="{ 'oc-pr-s': !isModifiable }" class="details-buttons">
+      <span class="expiration-date-hint" v-text="expirationDateHint" />
```

Comment:
remove ?

Reply:
huh why is this there again, i don't get it